sklearn/compose/tests/test_column_transformer.py

```diff
@@ -480,6 +480,9 @@ def test_column_transformer_mixed_cols_sparse():
         ct.fit_transform(df)
 
 
+# TODO(1.4): Remove filterwarning when `sparse` is replaced by`sparse_output`
+# in OneHotEncoder.
+@pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
You should not ignore all `FutureWarning`. You should match partially the deprecation warning otherwise, we might suppress other `FutureWarning` and not catch that something is wrong.

Reply:
Resolved almost all issues pointed out. The annoying case that you mentioned is the only thing left.
Thanks for the quick review!